exercises/resistor-color-duo/ResistorColorDuo.cs

```diff
@@ -0,0 +1,11 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
```

Comment:
These namespaces don't have to be used by the student, so we can remove them here.

```suggestion
```